### ISSUE TITLE
Consistently put a reference for Items

### ISSUE THREAD

**Issue by skaldarnar** [potsdam-pnp/pf2e-cards#39]
## Goal

Find a good place for source book references for the subject of a card itself and apply that consistently.

Remaining tasks:

- [x] allow for reference on `\Card` macro
- [x] add references to all cards
- [ ] make the arguments mandatory
- [ ] harmonize order of arguments in \Card and \Action

## Context

For many cards (like spells and cantrips) giving a source book reference on the respective action is just fine.

However, for Items (like Breastplate or Dagger) or cards explaining Rules or Conditions we want to provide a reference for the item itself.

On some cards we add a reference in the footer section mentioning other item properties aligned to the right-hand side:

https://github.com/potsdam-pnp/pf2e-cards/blob/f92fdcc6b39e3554c963108f10c37072181969a4/cards/items/breastplate.tex#L30

On other cards, this reference is completely missing.
In addition, the footer row is already pretty crowded for items like the Crossbow, with no obvious space left for the reference.

<img width="49%" src="https://github.com/potsdam-pnp/pf2e-cards/assets/1448874/9b866495-d94a-4e6a-878b-8fb78de31901"/>
<img width="49%" src="https://github.com/potsdam-pnp/pf2e-cards/assets/1448874/7a9ac338-7328-4f4f-8179-b1dc36e23484"/>

--- 

Observed on f92fdcc6b39e3554c963108f10c37072181969a4

**Comment by fkz** [comment]
Yes, I introduced it on the footer line for items where there was still enough space in 6b81fb9, which is not the case for weapons. My reasoning was that it's better to have it sometimes than to not have it at all.

**Comment by fkz** [comment]
Also, is this really only limited to items? We should have a reference to the book on every card, I think. For spells, the correct reference is already in the "Cast" action though.

**Comment by Toxaris** [comment]
We changed the layout of the cards to make room for a reference on every card in c428f9bee29da65aa98068de9a541468ea1a0678.

![image](https://github.com/potsdam-pnp/pf2e-cards/assets/961451/0799ba71-ca5c-451b-9b27-d6bcd2f1d698)

**Edit(skaldarnar):** Moved _Remaining Tasks_ to OP.
